docs/example.md

```diff
@@ -54,9 +54,9 @@ You should see a collection of virtual services, virtual nodes and a mesh, along
 
 Verify App Mesh is working
 ```
-kubectl run -n appmesh-demo -it curler --image=tutum/curl /bin/bash
+kubectl run -n appmesh-demo -it curler --image=pstauffer/curl sh
 
-for i in {1..100}; do curl colorgateway:9080/color; echo; done
+for i in 1 2 3 4 5 6 7 8 9 10; do curl colorgateway:9080/color; echo; done
```

Comment:
Is there a reason you changed from 100 to 10 iterations?

Reply:
Have updated it to a new format. The original statement runs does curl only once instead of a loop of 100.